Automattic/wp-calypso
# Jetpack Pricing Page: Simplify display of the realtime-only products

#### Changes proposed in this Pull Request

This PR updates the pricing page so that the new realtime-only products no longer display with tier 1 and 2 plans being separate cards, but instead shows a single "Backup" and "Security" card that adjust based on what the user owns.

Example with no owned products:
![image](https://user-images.githubusercontent.com/42627630/134572875-8063ebc2-20fe-40a8-a3ae-87eb7d26eab3.png)

Example with a owned product:
![image](https://user-images.githubusercontent.com/42627630/134573027-0d68255a-14df-4119-b551-ffb1f189871a.png)

#### Testing instructions



1. For a given site, visit `http://jetpack.cloud.localhost:3001/pricing/:site_slug?flags=jetpack/only-realtime-products`.
2. Check that page using each of the new realtime products (Backup and Security Tier 1 & 2) as well as Jetpack Complete, and verify that the correct cards display on the page for each case.
3. Visit both `http://jetpack.cloud.localhost:3001/pricing/:site_slug` and `http://jetpack.cloud.localhost:3001/pricing` to ensure the existing pricing page has not changed.



Related to 1200412004370260-as-1201041449585195
